identity/src/main/java/com/stripe/android/identity/ui/IndividualWelcomeScreen.kt

```diff
@@ -0,0 +1,217 @@
+                        },
+                    fontSize = 24.sp,
+                    fontWeight = FontWeight.Bold
+                )
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                    modifier = Modifier.padding(bottom = 12.dp)
+                ) {
+                    Image(
+                        painter = painterResource(id = R.drawable.time_estimate_icon),
+                        contentDescription = stringResource(id = R.string.description_time_estimate)
+                    )
+                    Html(
+                        html = individualWelcomePage.timeEstimate,
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(start = 6.dp)
+                            .semantics {
+                                testTag = INDIVIDUAL_WELCOME_TIME_ESTIMATE_TAG
+                            },
+                        color = MaterialTheme.colors.onBackground.copy(alpha = 0.6f),
+                        urlSpanStyle = SpanStyle(
+                            textDecoration = TextDecoration.Underline,
+                            color = MaterialTheme.colors.secondary
+                        )
+                    )
+                }
+                Html(
+                    html = individualWelcomePage.privacyPolicy,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = dimensionResource(id = R.dimen.item_vertical_margin))
+                        .semantics {
+                            testTag = INDIVIDUAL_WELCOME_PRIVACY_POLICY_TAG
+                        },
+                    color = MaterialTheme.colors.onBackground.copy(alpha = 0.6f),
+                    urlSpanStyle = SpanStyle(
+                        textDecoration = TextDecoration.Underline,
+                        color = MaterialTheme.colors.secondary
+                    )
+                )
+                Divider(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = dimensionResource(id = R.dimen.item_vertical_margin))
+                )
+
+                Html(
+                    html = individualWelcomePage.body,
+                    modifier = Modifier
+                        .padding(bottom = dimensionResource(id = R.dimen.item_vertical_margin))
+                        .semantics {
+                            testTag = INDIVIDUAL_WELCOME_BODY_TAG
+                        },
+                    color = MaterialTheme.colors.onBackground,
+                    urlSpanStyle = SpanStyle(
+                        textDecoration = TextDecoration.Underline,
+                        color = MaterialTheme.colors.secondary
+                    )
+                )
+            }
+
+            var acceptState by remember { mutableStateOf(LoadingButtonState.Idle) }
```

Comment:
Generally its a good practice to have the state at the beginning the composable.